XamBasePacket/Converters/NullToBoolConverter.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using Xamarin.Forms;
+
+namespace XamBasePacket.Converters
+{
+    public class NullToBoolConverter : IValueConverter
+    {
+        public bool WhenNull { get; set; }
+
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (value is string str)
```

Comment:
I think we might want to add one more clause for collections, something like
`if(value is IList/ICollection collection)
return collection == null || !collection.any() ? WhenNull : !WhenNull`

Reply:
Added